### PR TITLE
Support <skip> configuration for PMD, too

### PR DESCRIPTION
**Pull request by erwint**
This one was a bit more complicated, as there are two Executions (check and forked pmd). The skip needs to be checked on the "check" execution, other config is read from "pmd".

### CODE REVIEW

File: com.basistech.m2e.code.quality.pmd/src/main/java/com/basistech/m2e/code/quality/pmd/EclipsePmdProjectConfigurator.java [hunk 1]
```diff
@@ -107,12 +108,20 @@ protected void handleProjectConfigurationChange(
 		final MavenPluginConfigurationTranslator pluginCfgTranslator =
 		        MavenPluginConfigurationTranslator.newInstance(maven,
 		                session, mavenProjectFacade.getMavenProject(monitor),
-		                pmdGoalExecution, project, monitor);
-
+		                execution, pmdGoalExecution, project, monitor);
 		this.createOrUpdateEclipsePmdConfiguration(mavenPluginWrapper, project,
 		        pluginCfgTranslator, monitor, session);
-
+
```
Comment (victornoel) [inline com.basistech.m2e.code.quality.pmd/src/main/java/com/basistech/m2e/code/quality/pmd/EclipsePmdProjectConfigurator.java]: careful you introduced some whitspace on an empty line :)

Comment (vorburger) [reply com.basistech.m2e.code.quality.pmd/src/main/java/com/basistech/m2e/code/quality/pmd/EclipsePmdProjectConfigurator.java]: @victornoel if you're into this sort of thing, fancy contributing a Checkstyle failing the build for WS ?

Comment (victornoel) [reply com.basistech.m2e.code.quality.pmd/src/main/java/com/basistech/m2e/code/quality/pmd/EclipsePmdProjectConfigurator.java]: @vorburger should we start with a minimal setup with forbidding such thing or do you know of a checkstyle configuration that we could use as a starter?

Comment (vorburger) [reply com.basistech.m2e.code.quality.pmd/src/main/java/com/basistech/m2e/code/quality/pmd/EclipsePmdProjectConfigurator.java]: let's do this in #156

**Pull request merged**